.readthedocs.yaml

```diff
@@ -1,14 +1,5 @@
 version: 2
 
-build:
-  os: "ubuntu-20.04"
-  tools:
-    python: "3.10"
-  jobs:
-    post_build:
-      - find _build/html
```

Comment:
here, @mpryc changed in https://github.com/konveyor/pelorus/pull/812/files to make job work again

Do you think we need to change it more?

Reply:
I this was just post build step to list all generated files. I was thinking about removing it all together as it's mostly for debug the failures in CI. So I am fine removing it.